docs/SetupGuide_Javascript.md

```diff
@@ -110,23 +108,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `select * from Products where Cost = @Cost` query, returning the result as json object in the body. The *parameters* attribute passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. *commandType* is set to `Text`, since the constructor argument of the binding is a raw query.
```

Comment:
It would be better to move these descriptions into the functions themselves and then this can just be a link to the function folder. That way if someone is looking at the samples directly they can also see this useful description. 

(this goes for all our samples)